src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -298,14 +298,16 @@ def Main():
   # keep first occurrence only.
   python_path_entries = [
     GetWindowsPathWithUNCPrefix(d)
-    for d in Deduplicate(python_path_entries)
+    for d in python_path_entries
   ]
 
   old_python_path = os.environ.get('PYTHONPATH')
   python_path = os.pathsep.join(python_path_entries)
   if old_python_path:
     python_path += os.pathsep + old_python_path
 
+  python_path = os.pathsep.join(Deduplicate(python_path.split(os.pathsep)))
+
```

Comment:
Current implementation with joining, splitting, joining, is not the most readable.

Perhaps rewrite this block, by adding old_python_path to python_path_entries and then deduplicate and join with path separator.